docs/application/native/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1186 @@
+    -   The recognition result is invoked by the registered event handler.
+    -   You can get a matched command list from the voice control engine.
+    -   You can select command among matched commands.
+
+-   Retrieving information
+    -   You can [get various information](#info) from the voice control manager:
+        -   Voice control manager state
+            -   The state is changed by function calls and applied as a precondition for each API.
+        -   Voice control service state
+            -   The voice control service states are controlled by starting and stopping command recognition.
+        -   Current language
+            -   You can get the current language that the voice control engine uses as a base to recognize your utterance.
+            -   Only the commands based on the current language can be recognized. Therefore, your utterance must be in the current language.
+            -   The current language can be changed in application settings or by changing the language on the device display.
+        -   Supported language
+            -   You can retrieve a list of supported languages to check whether the language that you want is supported.
+
+
+To use the voice control manager:
+
+1.  Initialize the voice control manager and [register callbacks](#callback).
+
+    The initialization allows the voice control manager to distinguish your application from any other application that also uses voice control. Only one voice control manager application can work on the device. Therefore, if another application including voice control manager exists, your application cannot work properly on the same device.
+
+    The registered callbacks can be notified of the changes in service status, current language, recognition results, and errors.
+
+2.  Prepare the voice control manager
+
+    The preparation connects the Voice control service for background work, such as recording and recognizing the user voice.
+
+    When the application initializes and prepares the voice control manager, the voice control daemon is invoked and connected for background work. The daemon and the application communicate as server and client.
+
+2.  Set commands.
+
+    You can [create a command list](#commands), and add or remove individual commands in the list. While creating an individual command, set the command text and type for each command handle. When all the commands are created and added to the command list, set the command list to the voice control manager for recognition.
+
+3.  Prepare the voice control manager.
+
+    The preparation connects the voice control service. When the connection is complete, the service state changes to `Ready`.
+
+    When the application initializes and prepares the voice control manager, the voice control service is invoked and connected for the background work. The service and the application communicate as server and client.
+
+4.  Get the recognized command result and ASR.
```

Comment:
Is it like this ? 
4.  Get the recognized command result with Automatic Speech Recognition (ASR).

Reply:
Yes, the result includes commands list matching to the recorded utterance and the text result from ASR.

That is,
recognzied command result = commands list
ASR = text result from ASR